<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MountainFilterWidget</class>
 <widget class="QWidget" name="MountainFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="5">
    <widget class="QDoubleSpinBox" name="dsbBendFreqMin">
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QDoubleSpinBox" name="dsbBendFreqMax">
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QDoubleSpinBox" name="dsbBendAmountMax">
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QDoubleSpinBox" name="dsbMultMax">
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Seed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="dsbLengthMin">
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Width</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Bend Freq</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="sbRangesMax">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QDoubleSpinBox" name="dsbWidthMin">
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="sbRangesMin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Bend Amnt</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Scale</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>N Ranges</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="sbSeed">
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QDoubleSpinBox" name="dsbWidthMax">
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QDoubleSpinBox" name="dsbScaleMin">
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QDoubleSpinBox" name="dsbMultMin">
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QDoubleSpinBox" name="dsbScaleMax">
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QDoubleSpinBox" name="dsbBendAmountMin">
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Length</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Mult</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QDoubleSpinBox" name="dsbLengthMax">
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mountain Rage Filter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pbView">
     <property name="text">
      <string>View</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="pbDelete">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
